contracts/mainnet/connectors/liquity/main.sol

```diff
@@ -238,28 +245,27 @@ abstract contract LiquityResolver is Events, Helpers {
             lowerHint
         );
         
-        // Allow other spells to use the withdrawn collateral
-        setUint(setWithdrawId, withdrawAmount);
+        setUint(setIds[0], depositAmount);
+        setUint(setIds[1], withdrawAmount);
+        setUint(setIds[2], borrowAmount);
+        setUint(setIds[3], repayAmount);
 
-        // Allow other spells to use the borrowed amount
-        setUint(setBorrowId, borrowAmount);
-
-        _eventName = "LogAdjust(address,uint256,uint256,uint256,uint256,uint256,uint256,uint256,uint256,uint256)";
-        _eventParam = abi.encode(msg.sender, maxFeePercentage, depositAmount, withdrawAmount, borrowAmount, repayAmount, getDepositId, setWithdrawId, getRepayId, setBorrowId);
+        _eventName = "LogAdjust(address,uint256,uint256,uint256,uint256,uint256,uint256[],uint256[])";
+        _eventParam = abi.encode(address(this), maxFeePercentage, depositAmount, withdrawAmount, borrowAmount, repayAmount, getIds, setIds);
```

Comment:
Same here, feels like they should be using `adjustTrove.*` values

Reply:
Done https://github.com/liquity/dsa-connectors/pull/2/commits/d45f08b7430350a24555b2cd96b5834c72fd84c0